src/libs/routes.js

```diff
@@ -45,11 +47,13 @@ const routes = _.flatten([
   Dashboard.navPaths,
   Data.navPaths,
   Notebooks.navPaths,
+  Analysis.navPaths,
```

Comment:
`Analyses` instead of `Analysis`, analogously to `Notebooks`?

Reply:
Ah, the tab name was incorrect as `Analysis` too, I updated all references to `Analysis` to `Analyses` where applicable